### PR TITLE
More improvements

### LINKED ISSUE #11: Option to show all pages in the pagination
Before on issues.piwik.org we could show all pages listed in the pagination control. This is useful in our case, as we want to have a copy of all issues, and also we want google/etc to index all pages. If we use the pagination showing only a few links to pages, Google/etc likely wont index all pages as it requires too many clicks. 

Let's add a new config setting eg. `PAGINATION_COUNT_PAGES_TO_SHOW` which could default to 5 or 10, but in our case we would set it to 100 or 500  (it does not look very pretty of course, but it is functional in having search bots index all content without giving preference to the first pages)

### PR DESCRIPTION
**Pull request by Findus23**
Fixes #11 

Took a bit longer than expected as I left my notebook where I have everything set up at another house and immediadly after getting it back left the charger there 😭 

So please look over the changes before deployment, I haven't tested it much apart from "works for me".

### CODE REVIEW

File: src/templates/layout.twig [hunk 1]
```diff
@@ -22,6 +22,9 @@
         <div class="col-md-6">
             Powered by <a target="_blank" href="https://github.com/piwik/github-issues-mirror">GitHub Issue Mirror</a>
         </div>
+        <div class="col-md-6 text-right">
+            <a href="https://piwik.org/privacy-policy/">Privacy Policy</a>
```
Comment (mattab) [inline src/templates/layout.twig]: URL should be configurable in the config I suppose since other people use this project?

Comment (Findus23) [reply src/templates/layout.twig]: Indeed. I didn’t think about this. I’ll change it tomorrow

**Comment by Findus23** [comment]
@tsteur 
When #13 is merged, can you (or someone else) maybe take a look at this pull request?

The privacy policy in the footer and the sitemap are the major changes.

File: src/templates/layout.twig (hunk 1 above)
Comment (Findus23) [reply src/templates/layout.twig]: Tomorrow is a bit late, but I fixed it in https://github.com/matomo-org/github-issues-mirror/pull/12/commits/1154541d65c5ee61825417e93c35074f54efb1b9

**Comment by tsteur** [comment]
@Findus23 I'm getting an error that `/css/style.css` cannot be found. Is the file missing or my setup wrong?

**Comment by Findus23** [comment]
@tsteur 
That file needs to be compiled (I mentioned it in the readme)

**Comment by tsteur** [comment]
Can we maybe push that file? Saves us from having to compile it on the server and guarantees we all use / test the same version etc. Makes things a bit easier etc.

**Comment by Findus23** [comment]
While the package-lock.json should also guarantee that, I've committed it now. (And updated bootstrap to the stable version)

**Comment by tsteur** [comment]
👍 looks all good and thanks for committing. I know npm does as well, it just makes it much easier for everyone to commit it. Let me know if I should deploy it and adjust config file.

**Comment by Findus23** [comment]
If it works for you and you have time you can deploy it.
BTW: https://issues.piwik.org/ and http://issues.matomo.org/ don't seem to redirect

**Comment by tsteur** [comment]
Deployed and redirect is setup :+1: Cheers looks great

**Comment by tsteur** [comment]
It works now... it only needed to run the cronjob first

**Pull request merged**